sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/tracing.properties

```diff
@@ -1,3 +1,6 @@
 management.tracing.enabled=true
 management.tracing.propagation.type=b3,w3c
-management.tracing.sampling.probability=1.0
\ No newline at end of file
+management.tracing.sampling.probability=1.0
+management.tracing.grpc.endpoint=http://grafana-agent-traces.monitoring:4318
+management.tracing.grpc.compression=gzip
+management.tracing.grpc.timeoutMs=10000
```

Comment:
Should we describe these parameters [here](https://github.com/SDA-SE/sda-spring-boot-commons/blob/spring-boot-3/docs/starter-web.md?plain=1#L29) as well?

Not sure if others feel the same, but I'm always quite confused by all these tracing options. Should we provide example configs for common environments [here](https://github.com/SDA-SE/sda-spring-boot-commons/blob/spring-boot-3/docs/starter-web.md?plain=1#L466)?